restraint.spec

```diff
@@ -341,6 +345,7 @@ fi
 
 %files rhts
 %defattr(-,root,root,-)
+%attr(0644, root, root) %config(missingok) %{_sharedstatedir}/restraint/install_config
```

Comment:
I don't think that `missingok` is correct here. For reference, https://docs.fedoraproject.org/en-US/packaging-guidelines/#_configuration_files

Also, please, keep the format used in the file.

Reply:
I followed the syntax from the following link.  Not sure how else to do this.
https://serverfault.com/questions/691185/rpmbuild-spec-file-how-to-have-files-only-used-during-install